.github/workflows/update-license.yaml

```diff
@@ -0,0 +1,26 @@
+name: Update copyright year(s) in license file
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 1 1 *"
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: |
+            README.md
+            License
+          transform: (?<=[Copyright © |Copyright (c) ])(?<from>\d{4})?-?(\d{4})?
+          commitTitle: update my license
```

Comment:
```suggestion
          commitTitle: docs: update copyright years
```

Reply:
we need to change `docs:` to `docs/` because `docs:` lead to this error 
```
Nested mappings are not allowed in compact mappingsYAML
Incorrect type. Expected "string | number | boolean"
```

Reply:
Good point, thanks!  How about: 

```
commitTitle: "docs: update copyright years"
```

Reply:
Sure, I like it. let me test it